code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -980,6 +980,13 @@
 	..()
 	return
 
+/datum/reagent/sugar/overdose_process(var/mob/living/M as mob)
+	if(volume > 200)
+		M << "<span class = 'danger'>You pass out from hyperglycemic shock!</span>"
```

Comment:
Ah yes, because I know when I have hyperglycemia innately, instead of just blacking out.